src/test/scripts/product_versions.py

```diff
@@ -45,6 +53,13 @@ def get_lts_version(argv):
 	return lts_version
 
 
+def loadJSON(fdata):
+	result = re.search("\[(.*?)\]", fdata.decode("utf-8"))
```

Comment:
This regex will fail if there are any `]` in the objects, the `.*` pattern should probably be greedy here (drop the `?` at the end). This way this pattern will match from the first `[` to the last `]`